src/plugins/saved_objects/kibana.json

```diff
@@ -8,8 +8,5 @@
   "server": true,
   "ui": true,
   "requiredPlugins": ["data", "dataViews"],
-  "requiredBundles": [
-    "kibanaUtils",
-    "kibanaReact"
-  ]
+  "requiredBundles": ["kibanaUtils", "kibanaReact", "savedObjectsManagement"]
```

Comment:
The addition of a `SO` -> `SOM` dependency may be an issue. I see the upsides of using `getAllowedTypes` for the finder, but not sure if that's strictly necessary in the scope of adding tagging support to the component?